go.mod

```diff
@@ -1,12 +1,13 @@
 module github.com/stripe/skycfg
 
 require (
-	github.com/gogo/protobuf v1.2.0
-	github.com/golang/protobuf v1.2.0
+	github.com/gogo/protobuf v1.3.1
+	github.com/golang/protobuf v1.3.2
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
 	go.starlark.net v0.0.0-20190604130855-6ddc71c0ba77
-	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 )
 
 replace github.com/kylelemons/godebug => github.com/jmillikin-stripe/godebug v0.0.0-20180620173319-8279e1966bc1
+
+go 1.13
```

Comment:
It looks like this line is causing the CI build failure. Can you remove it, and also revert the change to `.travis.yml`?

Reply:
Yeah I was in the middle of making another PR for this when I saw your other change. Done.